ApiBundle/Transformer/ContentTransformer.php

```diff
@@ -75,7 +75,8 @@ public function transform($content)
             $facade->addAttribute($contentAttribute);
         }
 
-        if ($this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_UPDATE_CONTENT_TYPE_FOR_CONTENT)) {
+        if ($this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_CONTENT_TYPE_FOR_CONTENT)
+            && $this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_UPDATE_CONTENT_TYPE_FOR_CONTENT)) {
```

Comment:
I check it to disable the edit button in the dashboard widget. The UBO should not see the edit button if he have no access role nor update role. But you're right, it's not the best idea I had. I'm gonna fix it in the view.